subworkflows/local/deepvariant_caller.nf

```diff
@@ -5,20 +5,40 @@
 params.deepvariant_options = [:]
 
 include { DEEPVARIANT } from '../../modules/local/deepvariant/main'  addParams( options: params.deepvariant_options )
+include { GLNEXUS } from '../../modules/nf-core/modules/glnexus/main'  addParams( options: params.deepvariant_options )
 
 workflow DEEPVARIANT_CALLER {
     take:
         bam // channel: [ val(meta), path(bam), path(bai) ]
         fasta // path(fasta)
         fai // path(fai)
+        sample // channel: [ sample, sex, phenotype, paternal_id, maternal_id, case_id ]
 
     main:
         DEEPVARIANT ( bam, fasta, fai )
+        file_list = DEEPVARIANT.out.gvcf.collect{it[1]}.toList()
+
+        //retrieve case id for glnexus and store it in a new channel called case_meta
+        sample
+            .first()
+            .map{
+                it ->
+                    new_sample_meta = it.clone()
+                    new_sample_meta.id = new_sample_meta.case_id
+                    [ [ 'id':new_sample_meta.id ] ] }
+            .set {case_meta}
+
+        //Combine case meta with the list of gvcfs
+        case_meta.mix(file_list)
+            .collect()
+            .set { ch_gvcfs }
```

Comment:
Maybe this would work? :) @ramprasadn 

ch_gvcfs = case_meta.mix(file_list)

Reply:
Turns out collect is necessary. Without it, you don't get the items emitted as a list, instead, you get case_meta emitted first and then file_list, which doesn't work.